src/components/header/Header.tsx

```diff
@@ -111,8 +116,8 @@ const Header: React.FC<Props> = (_): JSX.Element => {
     <HeaderContainer>
       <>
         <Row justify="space-between" align="middle" style={{ height: headerHeight }}>
-          <Col xs={0} sm={0} md={0} lg={24}>
-            <Row justify="space-between" align="middle" style={{ height: headerHeight }}>
+          {(screens.lg || screens.xl || screens.xxl) && (
```

Comment:
[suggestion] Instead of checking all breakpoints, how about something like this:

```
const isDesktopView = screens.lg
...
{isDesktopView && (...)}
```